archunit/src/main/java/com/tngtech/archunit/lang/syntax/elements/MembersShould.java

```diff
@@ -107,6 +107,33 @@
     @PublicAPI(usage = ACCESS)
     CONJUNCTION haveFullNameNotMatching(String regex);
 
+    /**
+     * Asserts that members have a full name starting with given prefix.
```

Comment:
If I'm not mistaken, the new methods test member _names_, but not _full names_ (which would include their owners' FQCN).